product_docs/docs/biganimal/release/getting_started/activating_regions.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Activating regions"
+
+---
+
+When you activate a region, BigAnimal prepares the infrastructure required to deploy clusters. You must activate a cloud region before creating clusters in that region.
+
+!!! Note
+    BigAnimal might take a while to provision the infrastructure for a region. You can select a region for deploying your cluster even if it is not yet *Active*. Your cluster creation request is added to a queue, and the cluster will be created after your region is activated.
+
+## Prerequisite
+
+Your cloud service provider (CSP) must be set up with BigAnimal, see [Setting up your identity provider](../getting_started/identity_provider/)
+
+## Activate a new region
+
+Activating a new region can increase your cloud costs.
+
+1. On the left panel, under Admin, select **Regions**.
+
+1. A list of previously activated regions appears. Select **Activate New Region**. (Skip this step if this is your first time activating regions in your BigAnimal account.)
```

Comment:
It might be a little easier to parse if you use a format like:

1. This step is dependent on if you already have active regions or not:
    - If you already have regions activated ...
    - If this is your first time activating regions ...
    
 You might want to reverse the order of the sub bullets.

Reply:
Done.